README.md

```diff
@@ -130,6 +130,12 @@ logging:
   # regardless of the configured logging level.
   sampled_levels:
     'trace/webrequest': 0.2
+  # Set up the limiter for logging. When the rate of logs is close to the limit,
+  # it starts to increase the probability of discarding the logs of same kind.
+  # For default parameters use rate_limit: true
+  rate_limit:
```

Comment:
@Pchelolo, in the case of multiple ranges we could choose one of these semantics:

- Fancy: Apply them in level order. In this case, `error` and `fatal` could log 400/s, while `warn` would be limited at 200.
- Simple: Only allow one range, and throw an error if more were supplied.

Re keys, with quoting `'>=warn'` is perfectly legal.